client/validations/uniqueTechnicalUserName.directive.js

```diff
@@ -0,0 +1,31 @@
+angular.module('probedock.uniqueTechnicalUserNameValidation', [ 'probedock.api' ]).directive('uniqueTechnicalUserName', function(api, $q) {
+  return {
+    require: 'ngModel',
+    link: function($scope, element, attrs, ctrl) {
+      ctrl.$asyncValidators.uniqueUserName = function(modelValue) {
+
+        // If the name is blank there can be no name conflict with another user.
+        if (_.isBlank(modelValue)) {
+          return $q.when();
+        }
+
+        var userId = $scope.user ? $scope.user.id : null;
+
+        return api({
+          url: '/users',
+          params: {
+            name: modelValue,
+            organizationId: $scope.currentOrganization.id,
+            pageSize: 1
```

Comment:
Limit query to technical users.

**ALSO** modify the `uniqueUserName` directory to only query human users.